src/config/config-localhost.js

```diff
@@ -31,9 +31,10 @@ export const config = {
   /**
    * Core Network Related Data
    */
+  infuraAPIKey: import.meta.env.VITE_APP_INFURA_PROJECT_ID,
```

Comment:
store this in a const
```suggestion
const  infuraAPIKey: import.meta.env.VITE_APP_INFURA_PROJECT_ID || "",
```